demos/blockfactory/code_to_block_xml.js

```diff
@@ -0,0 +1,393 @@
+// Copyright 2017 Juan Carlos Orozco Arena
+// Apache License Version 2.0
+
+BlockFactory.manualEdit = function(){
+  BlockFactory.updateBlocksFlag = true // Avoid infinite update loop
+  BlockFactory.updateBlocksFlag2 = true
+  BlockFactory.updateLanguage()
+}
+
+window.jco_updateFactoryBlocks = function(){

```

Comment:
Can we put these methods under the BlockFactory namespace?

Google style issue: space between `()` and `{`. See https://google.github.io/styleguide/jsguide.html.

Also, we would greatly appreciate [JSDoc](http://usejsdoc.org/) style documentation on each function.  It can be short.